.github/workflows/security-scan.yaml

```diff
@@ -0,0 +1,32 @@
+name: Weekly security scan
+
+on:
+  schedule:
+  # Cron for every Monday at 9:12 UTC.
+  - cron: "12 9 * * 1"
+
+# Remove all permissions from GITHUB_TOKEN except metadata.
+permissions: {}
+
+jobs:
+  scan:
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: [main, release-1.10, release-1.9, release-1.8]
```

Comment:
This needs to be updated to our branches.

Reply:
Oh! Thanks for catching that!